Assets/LeapMotionModules/InteractionEngine/Scripts/InteractionBehaviours/InteractionBrushBone.cs

```diff
@@ -17,50 +19,23 @@ public class InteractionBrushBone : MonoBehaviour {
     /** The active InteractionManager. */
     public InteractionManager manager;
     /** This InteractiveBrushBone's RigidBody. */
-    public Rigidbody capsuleBody;
-    /** This InteractiveBrushBone's Collider. */
-    public CapsuleCollider capsuleCollider;
-    /** This InteractiveBrushBone's target position. */
+    public Rigidbody body;
+    /** This InteractionBrushBone's Collider. */
+    public Collider col;
+    /** This InteractionBrushBone's FixedJoint. */
+    public FixedJoint joint;
+    /** This InteractionBrushBone's Metacarpal FixedJoint. */
+    public FixedJoint metacarpalJoint;
+    /** This InteractionBrushBone's target position. */
     public Vector3 lastTarget;
 
-    // Once the brush becomes dislocated, it then remains dislocated until it
-    // stops triggering and then the _dislocatedCounter expires.
-    private const int DISLOCATED_BRUSH_COOLDOWN = 3;
-    private int _dislocatedCounter = DISLOCATED_BRUSH_COOLDOWN;
-
-    /** Changes the collider to react to collisions as a trigger. */
-    public void startTriggering() {
-      capsuleCollider.isTrigger = true;
-      _dislocatedCounter = 0;
-    }
-
-    /** Determines whether the brush bone should react to collisions as a trigger. */
-    public bool updateTriggering() {
-      if (_dislocatedCounter < DISLOCATED_BRUSH_COOLDOWN) {
-        if (++_dislocatedCounter == DISLOCATED_BRUSH_COOLDOWN) {
-          capsuleCollider.isTrigger = false;
-          return false;
-        }
-        return true;
-      }
-      return false;
+    public void DisableColliderTemporarily(float seconds) {
```

Comment:
For safety you should store the reference to the currently running coroutine so you can stop it if this method gets called while it is still running.  If you don't stop it, you might get unexpected results.